docs/workflows/phylogenetic_construction/clair3_variants.md

```diff
@@ -0,0 +1,134 @@
+# Clair3 Variants
+
+## Quick Facts
+
+| **Workflow Type** | **Applicable Kingdom** | **Last Known Changes** | **Command-line Compatibility** | **Workflow Level** |
+|---|---|---|---|---|
+| [Phylogenetic Construction](../../workflows_overview/workflows_type.md/#phylogenetic-construction) | [Bacteria](../../workflows_overview/workflows_kingdom.md/#bacteria), [Mycotics](../../workflows_overview/workflows_kingdom.md#mycotics), [Viral](../../workflows_overview/workflows_kingdom.md/#viral) | PHB vX.X.X | Yes | Sample-level |
```

Comment:
the current version of this is standalone and doesn't generate a phylogenetic tree so i would argue movng this to a standalone workflow type.

Reply:
this is a sister-workflow to snippy_variants so I'm for keeping those two together. Down the road we have plans for a phylogenetic construction workflow that leverages this one.

Reply:
i'm convinced!